pkg/services/ngalert/api/lotex_am.go

```diff
@@ -256,6 +256,11 @@ func (am *LotexAM) RoutePostAMAlerts(ctx *models.ReqContext, alerts apimodels.Po
 	)
 }
 
+func (am *LotexAM) RouteGetReceivers(ctx *models.ReqContext) response.Response {
+	// TODO: implement
+	return NotImplementedResp
```

Comment:
When it should be implemented? If it is not supported yet probably it does not make sense to create API for it?